templates/http/base/deployment.yaml

```diff
@@ -24,11 +24,19 @@ spec:
     spec:
       containers:
       - image:  {{values.appContainer}}
-        name: app-inference
+        name: app-interface
         envFrom:
         - configMapRef:
             name: {{values.name}}-model-config
         ports:
         - containerPort: {{values.appPort}}
         securityContext:
-          runAsNonRoot: true
\ No newline at end of file
+          runAsNonRoot: true
+      {%- if values.dbRequired %}
+      - image: {{values.dbContainer}}
```

Comment:
I thought we were going to run the DB container in a separate pod? FWIW, having the DB separate from the app pod means that the RHOAI workspace can interact with it

Reply:
let me check about deploying it by itself